Week 1 - Hello Microverse project

Rebelzob opened the pull request:
Features

- Added index.html and styles.css
- Setup Linters and run tests
- Added .gitignore folder with exceptions
- Practice the Github flow

Kossi-stack reviewed (changes requested):
## Hi @Rebelzob,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966](https://user-images.githubusercontent.com/81444338/170121142-a9f93bb7-38bb-45cc-8327-aab2501ef90a.gif)

## To Highlight

✅ Descriptive pull request message
✅ Documented your work professionally
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [x] Please make sure to use [this readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and please be sure to edit all of the sections to better suit your project in the current branch there is no readme file kindly add it.
- [ ] Kindly make sure to use [these linters](https://github.com/microverseinc/linters-config/tree/master/html-css) provided by Microverse and make sure they are set up correctly and they all pass with green checkmarks.
 
 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Rebelzob commented:
@Kossi-stack Thank for the review! I added the README and changed the .gitignore as you suggested, also the 3 linters are installed into the project.

Rebelzob commented:
@Kossi-stack I will make a new branch, I am having a bunch of errors on this one because of the change I made on the .github folder.

wayungi reviewed (changes requested):
Hi @Rebelzob , 

![image](https://user-images.githubusercontent.com/99656687/234200054-89d3fce4-7f5b-4e07-a92f-9fad90948091.png)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
 - [ ] Correct GitHub flow 👍🏾 
 - [ ] All linter checks are passing 👍🏾 

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] Good work adding the readme template. There are some parts of this document that should be present and parts that you should delete. Please find them listed below
 - [ ] Delete all instructions. they start with the `>` and appear gray. check image below
![inst](https://github.com/Rebelzob/Hello-Microverse-Project/assets/99656687/c61b52cd-b132-4794-ad7d-6f1fa039570b)
- [ ] Delete the extra information below the MIT link
![inf](https://github.com/Rebelzob/Hello-Microverse-Project/assets/99656687/84c27be5-a745-4d4c-b1d1-c1db25061789)

 - [x] Add a license file using [this guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository)

- [ ] Add the following missing sections as well
   - Contributing
   - Show your support
   - Acknowledgement

-  [ ] Under the getting started section, please add the following
  - Prerequisites
  - Usage
 
  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Rebelzob commented:
@wayungi thanks for the review! what should I do now since I corrected all your suggestions? should I request a new review? thanks in advance

shahadat3669 commented:
@Rebelzob You're welcome! If you have addressed all the suggestions and made the necessary corrections, it's a good idea to request a new review. with the same PR.

Rebelzob commented:
> @Rebelzob You're welcome! If you have addressed all the suggestions and made the necessary corrections, it's a good idea to request a new review. with the same PR.

@shahadat3669 its the same PR

shahadat3669 commented:
Yes. I am reviewing your new updates.

Alusp commented on README.md:
```diff
@@ -89,19 +121,49 @@ Clone this repository to your desired folder:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
> _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

- [x] [Required] Please as requested by the previous  Code Reviewer, Kindly remove this sentence as its just telling you the recommended License to use. :+1:

Alusp replied:
> # 📖 [your_project_name] <a name="about-project"></a>
> Describe your project in 1 or 2 sentences.

- [x] [Required] Please as requested by the previous reviewer, kindly replace the word `your_project_name` with your actual project name, for example `Hello Microverse` and replace this phrase `Describe your project in 1 or 2 sentences.` to your project description.

Alusp commented on README.md:
```diff
@@ -89,19 +121,49 @@ Clone this repository to your desired folder:
 ## 🔭 Future Features <a name="future-features"></a>
 
+
+
 > Describe 1 - 3 features you will add to the project.
 
+
 - [ ] **[new_feature_1]**
 - [ ] **[new_feature_2]**
 - [ ] **[new_feature_3]**
 
```
- [ ] [Optional] Please remove this phrase `> Describe 1 - 3 features you will add to the project.` and replace these words `[new_feature_1]` with the features the project will have as this Highlight the most important and distinctive features of your project without overwhelming the reader. :smile:

Rebelzob replied:
Hi @Alusp thanks for your reviews, I made the changes needed on the README

Alusp reviewed (changes requested):
Hi @Rebelzob, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![close](https://media.giphy.com/media/3ogwFKH8NPzxxIl2Cc/giphy.gif)

 ## Highlights

- [ ] Correct GitHub flow :white_check_mark: 
- [ ] All linter checks are passing :white_check_mark: 
- [ ] Fair descriptive PR and title :white_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Qoosim commented on README.md:
```diff
@@ -102,6 +149,8 @@ Clone this repository to your desired folder:
 
 This project is [MIT](./LICENSE) licensed.
```
- [OPTIONAL] Kindly fix the MIT License link. It is broken at this moment.

![license](https://github.com/Rebelzob/Hello-Microverse-Project/assets/26597432/099c1cc3-06a8-4d4e-aa8e-db5f7c1fa79f)

Qoosim reviewed (approved):
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @Rebelzob, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 

![](https://camo.githubusercontent.com/54a5d803a138b5a1a1242a607e4c1192c852cab8c591c1ddcaa33d06b51560f6/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f686c516559335062684833692f67697068792e676966)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
